plugins/dig-now.cpp

```diff
@@ -339,21 +472,27 @@ static bool dig_tile(color_ostream &out, MapExtras::MapCache &map,
         case df::tile_dig_designation::Channel:
         {
             DFCoord pos_below(pos.x, pos.y, pos.z-1);
+            // todo: does can_dig_channel return false?
```

Comment:
are you saying the conditions it checks for can never happen here?

Reply:
It is a todo. But I would infer from there being a designation, that it can be channeled.